client/src/App.js

```diff
@@ -3,27 +3,45 @@ import React, { Component } from 'react';
 import SavedList from './Movies/SavedList';
 import MovieList from './Movies/MovieList';
 import Movie from './Movies/Movie';
+import { BrowserRouter as Route } from 'react-router-dom';
```

Comment:
```suggestion
import { Route } from 'react-router-dom';
```
You're importing the wrong thing and that's why nothing shows up